themes/SuiteP/css/popup.scss

```diff
@@ -48,6 +62,10 @@
     text-transform: uppercase;
     padding: 0 0 0 30px;
   }
+  .h1, .h2, .h3, h1, h2, h3{
```

Comment:
So that the CSS is easy to maintain. Please do not stack the selectors. Please split them out. if you need to reduce code duplication then consider using a mixin. As this will produce the same output.

Example:
```

@mixin popupHeading() {
   margin-top: 10px;
   margin-bottom: 10px;
}

.h1 {
  @include popupHeading();
}

.h2 {
  @include popupHeading();
}
/* ... */
```